lighthouse-cli/test/smokehouse/seo/expectations.js

```diff
@@ -31,11 +31,14 @@ module.exports = [
       'is-crawlable': {
         score: true,
       },
+      'hreflang': {
+        score: true,
+      },
     },
   },
   {
-    initialUrl: 'http://localhost:10200/seo/seo-failure-cases.html?status_code=403&extra_header=x-robots-tag:none',
-    url: 'http://localhost:10200/seo/seo-failure-cases.html?status_code=403&extra_header=x-robots-tag:none',
+    initialUrl: 'http://localhost:10200/seo/seo-failure-cases.html?status_code=403&extra_header=x-robots-tag:none&extra_header=link:' + encodeURI('<http://example.com>;rel="alternate";hreflang="xx"'),
+    url: 'http://localhost:10200/seo/seo-failure-cases.html?status_code=403&extra_header=x-robots-tag:none&extra_header=link:' + encodeURI('<http://example.com>;rel="alternate";hreflang="xx"'),
```

Comment:
this is getting quite nasty. Maybe we should move headers to an external file (e.g. `seo-failure-cases.html?headers=seo-failure-headers.json`) or keep them here, but in a separate field (`headers: ['Link: bla', 'x-robots-tag:none']`)? In the second case we could pass them to the static-server e.g. in a header (which would be a very meta thing to do). @brendankenny WDYT?

Reply:
`url` is compared to the LHR's `url` property (to make sure redirects or whatever were captured correctly), so if the headers will be passed in as a query param to static server they'll have to be in the `url` expectations string too.

@paulirish just brought up that sending the headers as request headers won't really work since smokehouse invokes lighthouse with just a URL...there's no way to add request headers (until/if #2746 :)

This is a `js` file, so there's a few ways we could make this better:
- save parts of the URL to variables at the top of the file so they are just concatted here
- create some kind of `headersParam([])` function so that this could be `url: 'http://localhost:10200/seo/seo-failure-cases.html?' + headersParam(['Link: bla', 'x-robots-tag:none'])` or whatever
- use the URL constructor to create the string and rely on implicit `toString` or save `href` to a variable up top and invoke down here
- some combination of the above